packaging/deb/erigon/DEBIAN/postrm

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#
+###############
+# Remove erigon installs
+##############
+sudo rm -rf /lib/systemd/system/erigon.service
+sudo deluser erigon
+sudo systemctl daemon-reload
```

Comment:
sudos here, and no new line

are we sure that this systemctl and rm of the service is the correct solution here.

Finding the canonical solution from debian policy might be a good idea here

Reply:
Will address

Reply:
Updated